lib/grpc/errors.ts

```diff
@@ -0,0 +1,10 @@
+class GRPCError {
```

Comment:
I would prefer a `type` over a class and what about renaming that `type` to `WalliError` and using it for all errors? I created some in `lib/consts/errors.ts`. Also why aren't you using an error code?